### PR TITLE
A few new questions, some formatting, and some number cleanup

### PR DESCRIPTION
**Pull request by leesoh**

### CODE REVIEW

File: scenario.md [hunk 1]
```diff
@@ -1,17 +1,21 @@
 # Scenario Questions #
-*These questions have no right and wrong answer. They should allow the candidate to provide full responses. The interviewers here are not just looking for complete answers but also keeping track of HOW a candidate responds (rambles, gets distracted, doesn't answer the question, etc.).*
+
+o right and wrong answer. They should allow the candidate to provide full responses. The interviewers here are not just looking for complete answers but also keeping track of HOW a candidate responds (rambles, gets distracted, doesn't answer the question, etc.).*
```
Comment (WebBreacher) [inline scenario.md]: Looks like you may have removed the beginning of this paragraph. Can you fix (add the old content back) and resubmit?

File: open.md [hunk 2]
```diff
@@ -1,62 +1,75 @@
 # Open-ended Questions #
```
Comment (WebBreacher) [inline open.md:1]: No issues with this file changes you committed here.

File: knowledge.md [hunk 3]
```diff
@@ -1,47 +1,63 @@
...
-1. When running an nmap scan, what source port can you specify to scan from to commonly bypass firewall rules? *@jstnkndy*
-2. What kind of attack is ARP Spoofing considered and how could you leverage it on a penetration test? *@jstnkndy*
-3. Explain what NBNS poisoning is and how it can be leveraged on a penetration test. *@jstnkndy*
-12. Answer true or false and explain your answer: two-factor authentication protects against session hijacking. *@jstnkndy*
-13. Explain what happens when a connection from your webbrowser to a webserver is initiated on every OSI layer (as detailed as you like). *@dhauenstein*
+
+1. What are the phases of a penetration test? *@whereistehnarwhal, reddit*
+2. What is the difference between a risk assessment, a vulnerability assessment, and a penetration test? *@whereistehnarwhal, reddit*
+3. When running an nmap scan, what source port can you specify to scan from to commonly bypass firewall rules? *@jstnkndy*
+4. Construct an Nmap SYN scan that does not do DNS lookups, does not ping the hos, and only returns open ports for tcp/139 and tcp/445. *@whereistehnarwhal, reddit*
```
Comment (WebBreacher) [inline knowledge.md]: Please change "hos" to "host"

**Review by WebBreacher — CHANGES_REQUESTED** [review]
I left a couple comments for you. Please address the issues and then resubmit. Thanks for the work you put in here!

**Comment by leesoh** [comment]
Sorry about that. Changes in!

**Pull request merged**